### DISCUSSION TITLE
Chinese characters in Latex are offset from normal position

### DISCUSSION THREAD

**Discussion by Yucohny** [remarkjs/remark#1042] [Q&A]
I was trying to use `rehype-mathjax` with `react-markdown`  to render Latex. But there are some Chinese Characters in Latex, and after finishing rendering, Chinese characters in Latex are offset from normal position.
I wonder if this is a real problem? If yes, is there any solution. Or just something wrong with my code.
Thank you!

+ xxx.tsx
```typescript
import React from "react"
import ReactMarkdown from "react-markdown"
// @ts-ignore
import {Prism as SyntaxHighlighter} from 'react-syntax-highlighter'
// @ts-ignore
import {prism} from 'react-syntax-highlighter/dist/esm/styles/prism'
import remarkGfm from "remark-gfm"
import remarkMath from 'remark-math'
import rehypeMathjax from 'rehype-mathjax'

const MyMarkdown = (props: { children: any; }) => {
    const {children} = props
    return (
        <ReactMarkdown children={children}
                       remarkPlugins={[remarkMath, remarkGfm]}
                       rehypePlugins={[rehypeMathjax]}
                       components={{...}}
         />
    )
}

export default MyMarkdown
```

+ xxx.md
```markdown
$$
功率_{动态}\varpropto\frac12×容性负载×电压^2×开关频率
\\ 功率_{动态}\varpropto能耗_{动态}×开关频率
$$
```

**Comment by wooorm** [comment]
> I wonder if this is a real problem? If yes, is there any solution. Or just something wrong with my code.

I don’t know if (and where) it’s a problem!

I’d assume somewhere in mathjax though.
Or with CSS: that there are fonts missing?
Are you including the needed CSS?
And did you see the mathjax issues? https://github.com/search?q=org%3Amathjax+CJK+OR+Chinese+is%3Aissue&type=issues
